convert/jenkinsjson/json/readyaml.go

```diff
@@ -26,6 +29,8 @@ func ConvertReadYaml(node Node, variables map[string]string) *harness.Step {
 	var runCommand string
 	if file != "" {
 		runCommand = fmt.Sprintf("cat %s", file)
+	} else if text != "" {
+		runCommand = fmt.Sprintf("echo '%s'", text)
 	} else {
 		log.Printf("no valid attribute found for node %s", node.SpanName)
 		return nil
```

Comment:
This looks good, but here's a suggestion.  In Go, typically an early return is preferred for readability. It's a matter of style though, and this change keeps in with the existing style.

Here's an example of early return:

if file == "" && text == "" {
	log.Printf("no valid attribute found for node %s", node.SpanName)
	return nil
}

if file != "" {
	runCommand = fmt.Sprintf("cat %s", file)
} else {
	runCommand = fmt.Sprintf("echo '%s'", text)
}

Reply:
Thanks for the suggestion, I have refactored the function to follow this convention throughout